typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -979,6 +1008,37 @@ export const test3: ChainMetadata = {
   rpcUrls: [{ http: 'http://127.0.0.1:8545' }],
 };
 
+export const viction: ChainMetadata = {
+  blockExplorers: [
+    // {
```

Comment:
Why is this commented out? I think its blockscout based?

Reply:
I think the chain metadata check doesnt work against it

Reply:
Added it and it seems to work